Cannot run mvn command with ProcessBuilder on Windows

## Description
When I run `java -Dpaperclip.install=true -jar paperclip.jar` :
```
$ java -Dpaperclip.install=true -jar paperclip.jar
Maven must be installed and on your PATH.
java.io.IOException: Cannot run program "mvn": CreateProcess error=2, Le fichier spécifié est introuvable
        at java.lang.ProcessBuilder.start(Unknown Source)
        at io.papermc.paperclip.Paperclip.mavenInstall(Paperclip.java:260)
        at io.papermc.paperclip.Paperclip.setupEnv(Paperclip.java:103)
        at io.papermc.paperclip.Paperclip.main(Paperclip.java:48)
Caused by: java.io.IOException: CreateProcess error=2, Le fichier spécifié est introuvable
        at java.lang.ProcessImpl.create(Native Method)
        at java.lang.ProcessImpl.<init>(Unknown Source)
        at java.lang.ProcessImpl.start(Unknown Source)
        ... 4 more
```
It seems to be a problem with java + Windows.

This guy got the same issue :
https://stackoverflow.com/questions/7966194/running-mvn-using-processbuilder

When I tried to test `new ProcessBuilder("mvn.cmd", "-version").start()`, it worked perfectly. So if `mvn.cmd` can be automatically used instead of `mvn` for windows users, it would fix the thing I think.

Thank you in advance ! :D


## Additional Information
System path is correctly set for the maven bin so I can run `mvn -version` in console without any issues.

## System information
Java : jdk 1.8_0_201
OS : Windows 10 version 19041.450
16Go RAM, x64

--- a/java8/src/main/java/io/papermc/paperclip/Paperclip.java
+++ b/java8/src/main/java/io/papermc/paperclip/Paperclip.java
@@ -29,6 +29,7 @@ import java.nio.file.StandardCopyOption;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 import java.util.Arrays;
+import java.util.Locale;
 import java.util.jar.JarInputStream;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipFile;
@@ -256,8 +257,10 @@ public final class Paperclip {
     }
 
     private static void mavenInstall(Path paperJar) {
+        // On Windows we need to use "mvn.cmd" instead of "mvn"
+        final String mavenCommand = System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("windows") ? "mvn.cmd" : "mvn";
         try {
-            if (new ProcessBuilder("mvn", "-version").start().waitFor() != 0) {
+            if (new ProcessBuilder(mavenCommand, "-version").start().waitFor() != 0) {
                 // Error! It's either not on the path, or something went terribly wrong.
                 System.err.println("Maven must be installed and on your PATH.");
                 System.exit(1);
@@ -275,7 +278,7 @@ public final class Paperclip {
                 System.exit(1);
             }
 
-            if (new ProcessBuilder("mvn", "install:install-file", "-Dfile=" + paperJar, "-DpomFile=" + pomPath).start().waitFor() != 0) {
+            if (new ProcessBuilder(mavenCommand, "install:install-file", "-Dfile=" + paperJar, "-DpomFile=" + pomPath).start().waitFor() != 0) {
                 // Error! Could not install the file.
                 System.err.println("Could not install the Paper file.");
                 return;

